### ISSUE TITLE
Linter expect semicolons after goto 'Label:'

### ISSUE THREAD

**Issue by mkruglova** [tsqllint/tsqllint#97]
If goto statement is used lint expects ';' after referenced label: 

```
(50)		BEGIN CATCH
			.......
(55)			GOTO ERROR_HANDLER;
(56)		END CATCH;
		.....
(212)		ERROR_HANDLER:
(213)		BEGIN
(214)			    RAISERROR (@ErrorMessage, 11, 1);
(215) 	    		    RETURN; 
(216)		END;
``` 

Test.sql(212,19): error semicolon-termination : Statement not terminated with semicolon.

**Comment by dougwilson** [comment]
Yea, there shouldn't be a semicolon after a label.

**Comment by nathan-boyd** [comment]
resolved in 0031ea6a91344d06d4032496a7024087046860bd
